bundlewrap/cmdline/debug.py

```diff
@@ -28,6 +28,17 @@ def bw_debug(repo, args):
     if args['command']:
         exec(args['command'], env)
     else:
-        set_completer(Completer(env).complete)
-        parse_and_bind("tab: complete")
+        # read 'bw debug' history, if it exists
+        history_filename = join(repo.path, '.bw_debug_history')
+        if isfile(history_filename):
+            readline.read_history_file(history_filename)
+
+        # set up tab completion
+        readline.set_completer(Completer(env).complete)
+        readline.parse_and_bind("tab: complete")
+
+        # launch interactive debug session
         interact(banner=banner, local=env)
+
+        # save history
+        readline.write_history_file(history_filename)
```

Comment:
Not sure if we should call https://docs.python.org/3/library/readline.html#readline.set_history_length before saving history, and if yes what would be a good number of items to keep.

@vain @MaZderMind any opinions?